code/modules/paperwork/paper.dm

```diff
@@ -394,6 +394,62 @@
 /obj/item/paper/crumpled/muddy
 	icon_state = "scrap_mud"
 
+/**
+ * Sheet designed for blanks
+ * var/code -indicates the form of the document
+ * var/category - to sort documents by department categories.
+ */
+/obj/item/paper/blank
+	var/code 
+	var/category
+
+/obj/item/paper/blank/nt_cmd_ja
+	code = "NT-CMD-JA"
+	category = "Command"
```

Comment:
make the categories defines so you cant fuck up by doing a typo

Reply:
> make the categories defines so you cant fuck up by doing a typo

What's the best way to make them? Through #define?

Reply:
I redid the code according to your recommendation through the category definitions.

Reply:
dont make datums for that yes, just defines is fine

Reply:
Okay, I already did that.